.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: php
+
+sudo: false
+
+git:
+  depth: false
+
+php:
+  - '7.3'
+  - '7.4snapshot'
+  - 'nightly'
+
+env:
+  - DEPENDENCIES=""
+  - DEPENDENCIES="--prefer-lowest --prefer-stable"
+
+matrix:
+  allow_failures:
+  - php: '7.4snapshot'
+  - php: 'nightly'
+
+before_script:
+  - git checkout -b built-ci-branch
+  - composer update --prefer-dist $DEPENDENCIES
+
+script:
+  - vendor/bin/phpunit --coverage-text
+  - vendor/bin/psalm
+  - vendor/bin/phpcs
+  - vendor/bin/infection --min-msi=80 --min-covered-msi=100
```

Comment:
Adding `-log-verbosity=none` here will stop writing to log file, slightly reducing infection runtime

Reply:
Is there a CLI mutation reporter, btw? Would love to see the new mutations in CI: very useful for contributors that aren't familiar with the tool

Reply:
`-s` shows escaped mutations to the commandline after execution. 

There is not a logger that logs *everything* to the commandline.